scripts/lint.js

```diff
@@ -25,17 +25,30 @@ const TESTS = {
     const collector = (invalidEntries, icon, index, array) => {
       if (index > 0) {
         const prev = array[index - 1];
-        if (icon.title.localeCompare(prev.title) < 0) {
+        const compare = icon.title.localeCompare(prev.title);
+        if (compare < 0) {
           invalidEntries.push(icon);
+        } else if (compare === 0) {
+          if (prev.slug) {
+            if (!icon.slug || icon.slug.localeCompare(prev.slug) < 0) {
+              invalidEntries.push(icon);
+            }
+          }
         }
       }
       return invalidEntries;
     };
+    const format = icon => {
+            if (icon.slug) {
+                return `${icon.title} (${icon.slug})`;
+            }
+            return icon.title;
+    };
```

Comment:
```suggestion
    const format = icon => {
      if (icon.slug) {
        return `${icon.title} (${icon.slug})`;
      }
      return icon.title;
    };
```

Reply:
Won't be the last time I do that! 😆 I use tabs for indentation almost exclusively; easy to make mistakes when using spaces instead.